tests/integration/baseTestAPI.py

```diff
@@ -159,6 +162,21 @@ def setUpClass(cls):
 
             sess.commit()
 
+            # todo: refactor below when we figure out best way for enums in help tables
+            # Create lookup dictionaries
+            js = sess.query(JobStatus)
+            cls.jobStatusDict = {j.name: j.job_status_id for j in js.all()}
+            jt = sess.query(JobType)
+            cls.jobTypeDict = {j.name: j.job_type_id for j in jt.all()}
+            ft = sess.query(FileType)
+            cls.fileTypeDict = {f.name: f.file_type_id for f in ft.all()}
+            fs = sess.query(FileStatus)
+            cls.fileStatusDict = {f.name: f.file_status_id for f in fs.all()}
+            sev = sess.query(RuleSeverity)
+            cls.ruleSeverityDict = {s.name: s.rule_severity_id for s in sev.all()}
+            et = sess.query(ErrorType)
+            cls.errorTypeDict = {e.name: e.error_type_id for e in et.all()}
+
```

Comment:
@jworcestBAH Agreed that we need a better overall approach to handling these types of lookups. I didn't move `getIdFromDict` to the function bag because I would like to phase it out in favor of a lookup method that has less cognitive overhead, if you will, even if saves some extra db calls on these small tables.

Your comment, however, made me think harder about a way of having it both ways--minimize the db lookups while having a more straightforward way of looking up values. 

@cmc333333 pointed out that because our lookup values don't change as the app runs, we could create a `lookups.py` that has the values for each helper table. We could reference `lookups.py` in the db setup scripts, and the app itself could also use it.

What do you think?



Ideally, we could use postgres enums for these types of values, but I'm not sure how hard it is to implement those post-schema design, on an existing body of data.